### ISSUE TITLE
[A11Y] : A link present under Browser support section is labeled inappropriately

### ISSUE THREAD

**Issue by gkhadke** [ampproject/ampstart#391]
**ENVIRONMENT:**

OS (number): N (7.0)
Build number: NRD91R
Form factor (Smartphone / tablet):  Nexus 9
Browser: Chrome Dev 59.0.3062.4

**Pre-requisite:**
1. Talkback is ON

**STEPS TO REPRODUCE:**

1. Open [https://ampstart.com](https://ampstart.com)
2. Linearly Navigate to Sidebar Menu items and double tap on “How it works” link
3. Verify the links on the page by changing the talkback reading granularity level to “Links”. (Swipe one finger up or down to change the granularity to the desired level)

**ACTUAL RESULTS:**

While browsing through the links, user will encounter “here” link, which will have no meaning without context. In this scenario, purpose of using links granularity level fails for any type of screen reader.

**EXPECTED RESULTS:**

Link should be labeled such that, it should convey the purpose of that link even if it’s accessed independently. 

**ATTACHMENT(S):**

<img width="1440" alt="here link" src="https://cloud.githubusercontent.com/assets/26986019/24925685/da9e35ba-1ead-11e7-8ec0-ef881b8aae32.png">
